src/sqlfluff/rules/L010.py

```diff
@@ -87,19 +87,25 @@ def _eval(self, context: RuleContext) -> LintResult:
             cap_policy = self.cap_policy
             cap_policy_opts = self.cap_policy_opts
             ignore_words_list = self.ignore_words_list
+            ignore_words_regex = self.ignore_words_regex
```

Comment:
This is not exactly how this work.

You may not have noticed but we have the `ignore_words` config and ` ignore_words_list` attribute. Look similar but not quite the same, and the `ignore_words_list` are a split version of `ignore_words` config. So we do that once to cache it.

There is no processing needed for `ignore_words_regex`, but still think I'd call the attribute differently.

Or, since it needs no processing, you could just not have an attribute and delete this and call `getattr(self, "ignore_words_regex"))` directly on line 106 below? Though do need it twice so probably better to stick it into an attribute.

Reply:
I see. I'm not not 100% sure I understood correctly, but can you check latest commit to see if this is what you're saying?

Reply:
Yes that looks good now. Forgot you could use `self.ignore_words_regex` directly for configs/